### ISSUE TITLE
Odd header markup

### ISSUE THREAD

**Issue by rowanmanning** [pa11y/pa11y-site#8]
There's a `<p>` in the `<h1>` which is invalid. Got pointed out in a11y Slack :|

**Comment by rowanmanning** [comment]
No longer an issue, we have a new site now
